### ISSUE TITLE
How to use Russian Cyrillic letters in asciimath?

### ISSUE THREAD

**Issue by Thoragan** [pepijnve/asciimath#2]
I want to use Russian Cyrillic letters in [asciimath] blocks in Asciidoctor, but I noticed hat this causes asciimath to choke with error.

> invalid byte sequence in UTF-8

Here is simple example:
```
[asciimath]
++++
Скорость=(Расстояние)/(Время)
++++
```

**Comment by pepijnve** [comment]
The parser is not handling multi-byte characters correctly hence the problem. While reproducing this I've been able to fix the problem as well. I'll push that in a minute.

Note that

```
Скорость=(Расстояние)/(Время)
```
results in
```
<math><mi>С</mi><mi>к</mi><mi>о</mi><mi>р</mi><mi>о</mi><mi>с</mi><mi>т</mi><mi>ь</mi><mo>=</mo><mfrac><mrow><mi>Р</mi><mi>а</mi><mi>с</mi><mi>с</mi><mi>т</mi><mi>о</mi><mi>я</mi><mi>н</mi><mi>и</mi><mi>е</mi></mrow><mrow><mi>В</mi><mi>р</mi><mi>е</mi><mi>м</mi><mi>я</mi></mrow></mfrac></math>
```
which may not be what you expect. Since you're writing `speed = distance / time` (thank you Google translate) you probably want
```
"Скорость"=("Расстояние")/("Время")
```
which will get you
```
<math><mtext>Скорость</mtext><mo>=</mo><mfrac><mtext>Расстояние</mtext><mtext>Время</mtext></mfrac></math>
```
as output.

**Comment by pepijnve** [comment]
I've built and published asciimath 1.0.3 which contains this fix. Note that you'll need to use Ruby 1.9 for this to work properly. Ruby 1.8 did not have proper encoding support yet.

**Comment by Thoragan** [comment]
Thank you very much!
